components/dashboard/src/components/Alert.tsx

```diff
@@ -122,14 +123,14 @@ export default function Alert(props: AlertProps) {
             {props.closable && (
                 <span className={`mt-1 ml-4`}>
                     {/* Use an IconButton component once we make it */}
-                    <button
-                        type="button"
-                        className="bg-transparent p-1"
+                    <Button
```

Comment:
**issue(non-blocking)**: The button component here is causing a minor layout shift. Can be visible on large viewports or when the message in the alert is wrapper in a single line.

| CURRENT | IDEAL |
|-|-|
| <img width="1525" alt="Frame 1908" src="https://github.com/gitpod-io/gitpod/assets/120486/45a33f6d-eb47-4713-b096-785a682cf650"> | <img width="1525" alt="Frame 1909" src="https://github.com/gitpod-io/gitpod/assets/120486/8549a565-9423-4809-9845-91d517b122c2"> |